migrations/1572013310034_add-is-public-to-pool.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Pool } from '../src/Pool'
+
+const tableName = Pool.tableName
+const columnName = 'is_public'
+const columns = {
+  [columnName]: {
+    type: 'BOOLEAN',
+    default: true,
```

Comment:
Wouldn't it be better to just not this prop here and treat every pool like it's public?
Mostly because we're already doing this:
```
Pool.findOne<PoolAttributes>({ id })
```
here: https://github.com/decentraland/builder-server/pull/75/files#diff-64684767296fe331467d8529f62c5dacR58

Below we should do something like new Pool(utils.omit(project!, ['is_public'])).upsert() if we do go ahead with this option